lib/src/utils/path_utils.dart

```diff
@@ -88,6 +88,14 @@ String getFileExtension({
   }
 }
 
+String getFileTypeFromMime({required String contentType}) {
+  return contentType.substring(contentType.lastIndexOf('/') + 1);
+}
+
+String getBasenameWithoutExtension({required String filePath}) {
```

Comment:
@matibat , can you add the `withoutExtension` parameter and implement both options: with and without extension?

Reply:
I'd prefer to introduce a new method when needed. 🙏